packages/locale/typings/js-joda-locale.d.ts

```diff
@@ -10,6 +12,22 @@ export class Locale {
     public equals(other: any): boolean;
 }
 
+export class WeekFields {
+    public static ISO: WeekFields;
+    public static SUNDAY_START: WeekFields;
+    public static ofLocale(locale: Locale): WeekFields;
+    public static of(firstDayOrLocale: core.DayOfWeek | Locale, minDays?: number): WeekFields;
+    public static ofFirstDayOfWeekMinDays(firstDayOfWeek: DayOfWeek, minimalDaysInFirstWeek: number): WeekFields;
```

Comment:
LGTM, great work, also the tests! 

I am just wondering if we should keep this consistent and use `core.DayOfWeek` instead of `DayOfWeek`. Would this make the import above obsolete ?

Reply:
I am also in favor of moving the `import * as core from '@js-joda/core'` statement to the top and replace the other import.